src/amo/routes.js

```diff
@@ -47,6 +48,12 @@ export default (
         component={CollectionEdit}
       />
       <Route path="collections/:user/:slug/" component={Collection} />
+      {config.get('enableCollectionEdit') ?
```

Comment:
Why is adding under a feature flag (named editing) but not editing itself? Seems confusing.

Reply:
Kumar added the flag for enableCollectionEdit to hide the new editing screen, but we also want to hide the new creation screen for the same reason. I guess the flag is now misnamed, but I didn't think it made sense to create a second flag for adding. I'll rename it to something different that encompasses both.